slackbotExercise.py

```diff
@@ -463,6 +479,16 @@ def resetChallenges(bot):
 
     print "Reset users' challenges"
 
+def parseSlackJSON(response):
```

Comment:
Don't repeat this twice, I think we should make an `HTTP.py` class and import it in the main script and in `User.py`